README.md

```diff
@@ -18,79 +20,140 @@ Rather than installing and configuring many different build tools, such as Webpa
 - Vue.js projects
 - React projects
 
-## Example repositories using this configuration
-
-Refer to the demo folders in the [packages folder](https://github.com/DeloitteDigitalAPAC/webpack-config/tree/master/packages).
-
-## Packages included with Deloitte Digital Webpack Config
+## Table of contents
+
+- [Included Packages](#included-packages)
+	- [Core packages](#core-packages)
+	- [Vue.js packages](#vuejs-packages)
+	- [React packages](#react-packages)
+- [Installation](#installation)
+	- [Install for a standard js project](#install-for-a-standard-js-project)
+	- [Install for a Vue.js project](#install-for-a-vuejs-project)
+	- [Install for a React project](#install-for-a-react-project)
+	- [Configurating the tools](#configurating-the-tools)
```

Comment:
Broken link here, I think it should be to "Getting Started".

Reply:
Good catch